app/controllers/application_controller.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  helper_method :user_admin?
+
+  def user_admin?
+    current_user.role.eql? "admin"
```

Comment:
remove this..
instead add a `#admin?` method to user model
e.g.:
```
# instead of this
current_user.role.eql?('admin')
# to have this
current_user.admin?
```

Reply:
+1 and rename user_admin? => admin?

Reply:
Done, check please. And also, can your explain, what benefit of holding this method in user model instead of app conroller?

Reply:
just because you will use this method in many other places. you will see it later